library/core/src/mem/mod.rs

```diff
@@ -476,6 +476,8 @@ pub const fn align_of<T>() -> usize {
 #[stable(feature = "rust1", since = "1.0.0")]
 #[rustc_const_stable(feature = "const_align_of_val", since = "1.85.0")]
 #[allow(deprecated)]
+#[rustc_allow_const_fn_unstable(contracts)]
```

Comment:
I wonder if we should add `rustc_const_stable_indirect` to contract functions instead. @rust-lang/wg-const-eval?

Reply:
Which functions would that affect?

Reply:
I finally tried adding `#[rustc_const_stable_indirect]` to `contract_check_ensures`. Yet I then got:
```
error: const function that might be (indirectly) exposed to stable cannot use `#[feature(const_eval_select)]`
    --> library/core/src/intrinsics/mod.rs:2439:9
[...]
help: if the function is not (yet) meant to be exposed to stable const contexts, add `#[rustc_const_unstable]`
     |
2684 + #[rustc_const_unstable(feature = "...", issue = "...")]
2685 | pub const fn contract_check_ensures<C: Fn(&Ret) -> bool + Copy, Ret>(
     |
```
where the proposed fix won't actually work as the function is already marked with `rustc_const_unstable`.